src/core/symbology-ng/qgssymbollayerutils.cpp

```diff
@@ -2514,7 +2515,15 @@ bool QgsSymbolLayerUtils::functionFromSldElement( QDomElement &element, QString
     return false;
   }
 
+  // check if it is a single string value
+  if ( elem.hasChildNodes() &&
+       elem.firstChild().nodeType() == QDomNode::TextNode )
+  {
+    function = elem.firstChild().nodeValue();
```

Comment:
make sense... I supposed  the expressionFromOgcFilter would manage the parsing of an aexpression logic that is more complex. I'll give a check.